File: playground/BOJ/별찍기/boj_2439/boj_2439.js
/*
----------------------------------------------
Sub : [BOJ] 별찍기-2
Link: https://www.acmicpc.net/problem/2439
Tag : JS
Memo
----------------------------------------------
*/

const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

N = Number(input[0])

for (let i=0; i<N; i++) {
  console.log(' '.repeat(N-i-1) + '*'.repeat(i+1));
}

File: boj/boj_2439/boj_2439.py
'''
----------------------------------------------
Sub : [BOJ] 별찍기-2
Link: https://www.acmicpc.net/problem/2439
Tag : Python
Memo
----------------------------------------------
'''

N = int(input())

for i in range(1, N+1):
    print(' ' * (N-i), end='')
    print('*' * i)
